tests/my_test.py

```diff
@@ -10,42 +12,108 @@ def test_for_successful_response(self):
         # OR
         self.assertTrue('OK' == result.reason)
 
-
 class CareerPortalTests(unittest.TestCase):
     def setUp(self) -> None:
-        self.base_url = 'https://recruit-portnov.herokuapp.com/recruit/api/v1'
-        self.headers = {'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36'}
-
+        pass
+    '''
     def test_login(self):
-        positions = requests.get(self.base_url + '/positions')
+        sess = Authenticate() #sess is an object of class, creates instance of class
```

Comment:
sess is a variable pointing to an object, that is an instance of Authenticate class